lib/reader.js

```diff
@@ -44,5 +45,24 @@ module.exports = async (feeds, options) => {
         })
     );
 
-    return getStream(browserify({ entries, debug: opts.debug }).bundle());
+    const bundler = browserify({ entries, debug: opts.sourceMaps });
+
+    let envifyOptions = { ...process.env, _: 'purge' };
+    if (opts.minify) {
+        envifyOptions = { ...envifyOptions, NODE_ENV: 'production' };
+    }
+    bundler.transform('envify', envifyOptions);
+
+    if (opts.minify) {
+        bundler.transform('uglifyify', { global: true });
```

Comment:
can we use it? we still dedupe?

Reply:
No, we don't get dedupe like before but we'll get code removed in react. It's a bit of a trade-off but seems like the better option but its kinda guesswork without real bundles to test on.

Reply:
Why change, then? If we don't know which is better, I think we should measure before we change